index.html

```diff
@@ -492,6 +513,27 @@ <h3 class="card__title">Comments</h3>
         document.querySelector("input[name=siteUrl]").value = addProtocol(siteUrl);
       };
 
+      const toggleFormStatus = (e) => {
+        const isBrandedContentChecked = e.srcElement.classList.contains('branded-content-card');
+
+        if(isBrandedContentChecked){
+          const siteUrlInput = buttonsToDisable[0];
+
+          myForm.classList.add('form--blocked');
+          siteUrlInput.value = 'https://playground.mrf.io/templates/branded-content-example.html'
+        } else {
+          myForm.classList.remove('form--blocked');
+        }
+
+        for (let i = 0; i < buttonsToDisable.length; i++) {
+          buttonsToDisable[i].disabled = isBrandedContentChecked;
+        }
```

Comment:
Following my previous comments, I'd check here if the input has a `siteUrl` data attribute defined and disable the corresponding form fields if so.

I find it a bit weird having the URL field within a `buttonsToDisable` array, I'd simply disable the 2-3 required elements instead:

```
document.querySelector("#siteUrl").disabled = true;
//...
```
WDYT?